website_calendar_snippet/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+Calendar Snippet
+================
+
+This module adds a new block in the website editor.
+The new block is based on Messaging -> Calendar. Use field 'Privacy'
+to determine its publicity.
```

Comment:
To change privacy setting you should:
1. Click the event
2. Click **Edit Event** button
3. Go to **Options** tab

![website_calendar](https://cloud.githubusercontent.com/assets/1947376/13481142/a8ca5d1a-e115-11e5-86b9-9d947c9f12e5.png)